packages/utils/usePairTokensPriceInverted.ts

```diff
@@ -0,0 +1,40 @@
+export const usePairTokensPriceInverted = (
+  prices?: {
+    pairPriceData: {
+      time: Date
+      value: number
+    }[]
+    maxPrice: number
+    minPrice: number
+    averagePrice: number
```

Comment:
```
    maxPrice: number
    minPrice: number
    averagePrice: number
```
Any reason we cannot use this?

Reply:
Since the `prices` will not be inverted.